### PR TITLE
Fixed RunPostBuildEvent description

### LINKED ISSUE #246: Documentation Typo
I noticed the documentation for the `RunPostBuildEvent` MSBuild property contains incorrect info; I suspect a typo [[link to affected file](https://github.com/dotnet/roslyn-project-system/blob/master/docs/well-known-project-properties.md)]. Specifically, I believe the `OnOutputUpdated` and `OnBuildSuccess` rows in the tables have their descriptions backwards. See below screenshot for how it appears at time of this writing.

Documentation Typo

### PR DESCRIPTION
**Pull request by davkean**
Fixes #246

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/roslyn-analysis @wjk

**Comment by srivatsn** [comment]
👍

File: docs/well-known-project-properties.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ Specifies the conditions for the command in _PostBuildEvent_ to run.
 | Value           | Description    |
 |-----------------| ---------------|
 | Always          | Post-build event will run regardless of whether the build succeeds.     |
-| OnOutputUpdated | Post-build event will run if the build succeeds, regardless of whether the project output is up-to-date.|
-| OnBuildSuccess  | Post-build event will only when the project output is different than the previous output. |
+| OnOutputUpdated | Post-build event will only when the project output is different than the previous output.|
```
Comment (jinujoseph) [inline docs/well-known-project-properties.md]: Did you miss the work "Run" here ?

Comment (davkean) [inline docs/well-known-project-properties.md]: Yep, I just fixed it. Good catch!

**Pull request merged**